util/logger/config.go

```diff
@@ -1,14 +1,54 @@
 package logger
 
+import "path/filepath"
+
+const (
+	LogDirectory = "logs"
+	LogFilename  = "pactus.log"
+)
+
 type Config struct {
+	// ConsoleLoggingEnabled Enable console logging
```

Comment:
No new configuration for `log_to_file` and `file_rotation`. This file can be reverted.